packages/commonwealth/server/util/roles.ts

```diff
@@ -1,38 +1,35 @@
-import type { Transaction, FindOptions } from 'sequelize';
+import type { FindOptions, Transaction } from 'sequelize';
 import { Op } from 'sequelize';
-import { aggregatePermissions } from '../../shared/utils';
+import CommunityRole from '../../client/scripts/models/CommunityRole';
```

Comment:
Shouldn't ever import from client onto server -- move the CommunityRole file to shared/ if you REALLY need it, or construct it without typing.